features/supplier/brief_response.feature

```diff
@@ -147,3 +147,94 @@ Scenario: Supplier applies for a user-research-participants brief
       | Liking sugar lumps                 |                      |
       | Being good at jumping over fences  | No jump is too high. |
       | Saying "Neigh"                     | NEIGH                |
+
+Scenario: Previous page links are used during response flow
```

Comment:
Might be nice to add something in the scenario that indicates we are not just checking the links work but we are also checking that existing data is prefilled